#! /bin/sh -e

## DP: Small cosmetic changes for dovecot-sql.conf
## DP: Author: Jaldhar H. Vyas <jaldhar@debian.org>

. $(dirname $0)/DPATCH

exit 0
@DPATCH@
--- dovecot-1.2.0/doc/dovecot-sql-example.conf	2008-09-20 16:22:54.000000000 +0000
+++ dovecot-1.2.0-debian/doc/dovecot-sql-example.conf	2009-07-03 06:39:50.000000000 +0000
@@ -47,7 +47,7 @@
 #                           the default my.cnf location
 #     option_group        - Read options from the given group (default: client)
 # 
-#   You can connect to UNIX sockets by using host: host=/var/run/mysql.sock
+#   You can connect to UNIX sockets by using host: host=/var/run/mysqld/mysqld.sock
 #   Note that currently you can't use spaces in parameters.
 #
 #   MySQL supports multiple host parameters for load balancing / HA.
